src/Arduino_WiFiConnectionHandler.cpp

```diff
@@ -87,7 +87,7 @@ NetworkConnectionState WiFiConnectionHandler::update_handleInit()
   delay(300);
   WiFi.begin(_ssid, _pass);
   delay(1000);
-#endif /* ifndef BOARD_ESP8266 */
+#endif #if !defined(BOARD_ESP8266) && !defined(ESP32)
```

Comment:
I think this causes a compile issue for sure @ubidefeo ... can you please correct your PR?